### ISSUE TITLE
multidev build failure

### ISSUE THREAD

**Issue by alphex** [pantheon-systems/circleci-orb#24]
@stevector , thank you VERY much for this work, anything that makes learning CI/CD easier is hugely appreciated.

But, I hit a snag.

I created a branch, pushed to github, and made a pull request...

![image](https://user-images.githubusercontent.com/597641/59131783-4e3cd580-8941-11e9-9a8f-e328d1ffa36b.png)

When we get to the `Commit code to Pantheon repository and push to Pantheon` part of the job, I get this error message

![image](https://user-images.githubusercontent.com/597641/59131657-f4d4a680-8940-11e9-9c07-987c847d3ef2.png)

```
remote: PANTHEON NOTICE:        
remote: 
remote: Skipping code sync, no Multidev environments were found for branch "pr-1".        
remote: 
 [notice] Creating multidev pr-1 for site arsclan
 [notice] Created Multidev environment "pr-1"
 [notice] Enabled on-server development via SFTP for "pr-1"
 [error]  Could not figure out which git repository service to use with . 
```

I was watching the pantheon multidev dashboard, and I saw the PR-1 multidev being built, but then it vanished after this message, and the next 3 steps completed.

![image](https://user-images.githubusercontent.com/597641/59131727-277e9f00-8941-11e9-80db-5e274ca4d7d0.png)

The job says it was successful, and github's pull request says everything is fine.

![image](https://user-images.githubusercontent.com/597641/59131816-6f052b00-8941-11e9-84e2-51159f8e7066.png)

I do have a DEV, TEST and LIVE environment initialized, and any git pushes I was pushing right to the master branch worked perfectly.    

The ONLY wrinkle that I'm adding to this, is that I'm using the pantheon drops 8 with composer with no CI documentation to set up the code base.  https://pantheon.io/docs/guides/drupal-8-composer-no-ci/

But, again, pushing it to master worked, so, I hope thats not the problem.

Any thoughts anyone?   Thank you again!

**Comment by stevector** [comment]
Thanks for testing @alphex! I think I've seen this error but in a different context. Can you give me read access to the repo so I can review more closely?

**Comment by alphex** [comment]
@stevector you should have a collaborator invite.   Thank you.

**Comment by stevector** [comment]
Hi again @alphex! I was able to reproduce that message across my three repos that implement this orb.

But for all three of my testing repos the the multidev was still created. I also think the multidev was created for your site. I see it at `https://pr-1-SITEMACHINENAME.pantheonsite.io`.

The message is coming from inside the Build Tools Plugin, I can investigate where/why. I suspect that the issue is that the Orb is calling `terminus -n build:env:create` from a git clone of just the Pantheon repo. [Example Drops 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer) does this step a little differently by adding the Pantheon git repo as a second remote on the clone from github.

**Comment by alphex** [comment]
@stevector not sure what I did, or what you did, but yes, I do see the multidev!  Thank you.

I was out, and saw your message update on my phone, so I came home, and found the multidev screen where I left it. 

![image](https://user-images.githubusercontent.com/597641/59140747-df2aa580-896f-11e9-8544-453cd7f774d4.png)

So I hit refresh, and low and behold there was the multidev.

![image](https://user-images.githubusercontent.com/597641/59140749-ea7dd100-896f-11e9-9844-ce9b91eb5f77.png)

Is there a delay of some kind, in it showing up?   was my internet being stupid or my browser being stupid?   

Thank you for your help, Maybe this was just internet gremlins not reloading the page and I wasn't giving it enough patience on a friday evening.

I'll Close this, and thank you again.
